src/features/people/PeopleList/styled.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+
+export const SectionWrapper = styled.ul`
+	margin: 0;
+	padding: 0;
+	list-style-type: none;
+	display: flex;
+	flex-wrap: wrap;
+	gap: 24px;
+	width: 100%;
+
+	@media (max-width: 767px) {
```

Comment:
I would suggest to use references to theme.js (breakpoints - mobile) instead of @media (max-width: 767px).